### PR TITLE
Shinobi: Geeves Destroys Game Balance: Electric Boogaloo

### PR DESCRIPTION
**Pull request by Geevies**
Tweaks the ninja gamemode to spawn two ninjas instead of just one. To compensate for this, various tweaks to the gear ninjas get have been made.
As with all my PRs, I by no means consider this code to be finalized, as in a project like this, everything can be tweaked and changed, removed or added to after it gets merged. If anything is odd, fucks up, or makes you want to tear your hair out, contact me at Geeves#8863.

Additionally, I would like to thank BurgerBB for writing some of the code I took, as well as inspiring me to make this PR. Here is the PR I got his contribution to this PR from. #5833

Now, to the meat of this PR.

* Added a ninja frequency to radios which ninjas spawn with to collaborate.
* Two ninjas now spawn instead of just one.
* Gave the ninja better spawning gear, a syndicate uniform, a belt with some tools and a pAI card.
* The ninja suit now has new lights, and its shoes are now magboots.
* Added a doorhack RIG module. Functions the same as the normal doorhack, but the user can move around after initiating the hack, as well as hack multiple airlocks at once.
* Added new advanced combat belt, can hold various gear that a ninja would usually come across.
* Gave the ninja teleporter a 5 second cooldown between uses.
* Buffed the ninja's RIG armour values slightly. Still requires feedback, if ninjas die too quickly, it will be buffed more.
* Added an additional toolbox, chair, suitcooling unit and ninja spawn point to the ninja shuttle on the centcomm level.
* Gave the ninja a new uniform + gloves.
* Tweaked Emergency Power Generator to give 2500 energy, but take 5 minutes to recharge. In the past, they took 24 seconds to recharge and gave 1500 energy, which made all other sources of energy regeneration useless.

Reupload because branch histories a fuck.

### CODE REVIEW

File: code/game/antagonist/outsider/ninja.dm [hunk 1]
```diff
@@ -99,35 +99,20 @@ var/datum/antagonist/ninja/ninjas
...
-			player.internals.icon_state = "internal1"
-		else
-			to_chat(player, "<span class='danger'>You forgot to turn on your internals! Quickly, toggle the valve!</span>")
+	for (var/obj/item/I in player)
+		if (istype(I, /obj/item/weapon/implant))
+			continue
+		player.drop_from_inventory(I)
+		if(I.loc != player)
+			qdel(I)
+	player.preEquipOutfit(/datum/outfit/admin/syndicate/ninja/, FALSE)
```
Comment (NonQueueingMatt) [inline code/game/antagonist/outsider/ninja.dm]:
```suggestion
	player.preEquipOutfit(/datum/outfit/admin/syndicate/ninja, FALSE)
```

File: code/game/antagonist/outsider/ninja.dm [hunk 2]
```diff
@@ -99,35 +99,20 @@ var/datum/antagonist/ninja/ninjas
...
+	player.equipOutfit(/datum/outfit/admin/syndicate/ninja/, FALSE)
```
Comment (NonQueueingMatt) [inline code/game/antagonist/outsider/ninja.dm]:
```suggestion
	player.equipOutfit(/datum/outfit/admin/syndicate/ninja, FALSE)
```

File: code/modules/clothing/spacesuits/rig/suits/light.dm [hunk 3]
```diff
@@ -76,31 +76,40 @@
 /obj/item/weapon/rig/light/ninja
 	name = "stealth suit control module"
 	suit_type = "stealth suit"
-	desc = "A unique, vaccum-proof suit of nano-enhanced armor designed specifically for stealth operations."
+	desc = "A unique, vaccuum-proof suit of nano-enhanced armor designed specifically for stealth operations."
```
Comment (NonQueueingMatt) [inline code/modules/clothing/spacesuits/rig/suits/light.dm]:
```suggestion
	desc = "A unique, vacuum-proof suit of nano-enhanced armor designed specifically for stealth operations."
```

**Comment by NonQueueingMatt** [comment]
also icon conflicts

File: code/game/objects/items/weapons/storage/belt.dm [hunk 4]
```diff
@@ -347,6 +347,45 @@
...
+		/obj/item/weapon/plastique,
+		/obj/item/weapon/gun/projectile/pistol,
+		/obj/item/weapon/gun/energy/crossbow,
+		/obj/item/ammo_casing/a145,
+		/obj/item/device/radio/uplink,
+		/obj/item/weapon/card/emag,
+		/obj/item/device/multitool/hacktool,
+		/obj/item/weapon/reagent_containers,
+		/obj/item/stack/telecrystal,
+		/obj/item/weapon/material/sword,
```
Comment (Alberyk) [inline code/game/objects/items/weapons/storage/belt.dm]: Not sure if it is a good idea.

**Pull request merged**